common/src/main/java/org/apache/celeborn/common/meta/DiskStatus.java

```diff
@@ -34,4 +34,17 @@ public enum DiskStatus {
   public final byte getValue() {
     return value;
   }
+
+  public final String toMetric() {
```

Comment:
Why not just add a member variable to diskStatus directly, it will be more intuitive and  controllable

Reply:
I think this is fine, only to rename to metricName() instead of toMetric()